.github/workflows/GithubRunner.yaml

```diff
@@ -18,8 +18,15 @@ jobs:
       - name: Install dependencies
         run: |
           pip install .[tests]
-      - name: Run pytest
+          sudo apt install texlive-latex-recommended -y
+          sudo apt install ghostscript -y
+      - name: Run github runner pytest
         env:
           PYTEST_ADDOPTS: "--color=yes"
         run: |
           python -m pytest -v --github-runner
+      - name: Run pytest
+        env:
+          PYTEST_ADDOPTS: "--color=yes"
+        run: |
+          python -m pytest -v --time-tool time
```

Comment:
This isn't required, as the `Ubuntu` nad `Arch` workflows already run the tests with `--time-tool time`

Reply:
Sure. I was mislead by https://github.com/sio2project/sinol-make/blob/d229512b722af2a30311df43718dbaa9c0338664/tests/conftest.py#L37 so I also fixed it